content/swagger/resources/assets/routes/assets.yaml

```diff
@@ -0,0 +1,275 @@
+              ]  
+            },
+            {
+              "_links": {
+                "self": {
+                  "href": "https://demo.akeneo.com/api/rest/v1/assets/tshirt_artemis_maintenance_manual"
+                }
+              },
+              "code":"tshirt_artemis_maintenance_manual",
+              "categories": ["tshirts_assets"],
+              "description": "The maintenance manual of the Artemis t-shirt",
+              "localizable": true,
+              "tags": ["tshirts", "manuals"],
+              "end_of_use": null,
+              "variation_files": [],
+              "reference_files":[
+                {
+                  "_link": {
+                   "download": {
+                     "href": "https://demo.akeneo.com/api/rest/v1/assets/tshirt_artemis_maintenance_manual/reference-files/en_US/download"
+                    },
+                    "self": {
+                     "href": "https://demo.akeneo.com/api/rest/v1/assets/tshirt_artemis_maintenance_manual/reference-files/en_US"
+                    }
+                  },
+                  "locale": "en_US",
+                  "code": "7/5/8/e/759e39d48va7b42a55002434fd3d7b6cf3189b7f_tshirt_artemis_maintenance_manual-en_US.pdf"
+                },
+                {
+                  "_link": {
+                   "download": {
+                     "href": "https://demo.akeneo.com/api/rest/v1/assets/tshirt_artemis_maintenance_manual/reference-files/fr_FR/download"
+                    },
+                    "self": {
+                     "href": "https://demo.akeneo.com/api/rest/v1/assets/tshirt_artemis_maintenance_manual/reference-files/fr_FR"
+                    }
+                  },
+                  "locale": "fr_FR",
+                  "code": "7/5/8/e/759e39d48va7b42a55002434fd3d7b6cf3189b7f_tshirt_artemis_maintenance_manual-fr_FR.pdf"
+                }
+              ]  
+            }
+          ]
+        }
+      }
+    401:
+      $ref: "#/responses/401Error"
+    403:
+      $ref: "#/responses/403Error"
+    406:
+      $ref: "#/responses/406Error"
+post:
+  summary: Create a new asset
+  operationId: "post_assets"
+  description: This endpoint allows you to create a new asset.
+  tags:
+    - Assets
+  x-versions:
+    - 2.1
+  x-ee: true
+  parameters:
+    - name: body
+      in: body
+      schema:
+        $ref: "#/definitions/Asset"
+  responses:
+    201:
+      $ref: "#/responses/Created"
+    400:
+      $ref: "#/responses/400Error"
+    401:
+      $ref: "#/responses/401Error"
+    403:
+      $ref: "#/responses/403Error"
+    415:
+      $ref: "#/responses/415Error"
+    422:
+      $ref: "#/responses/422Error"
+patch:
+  summary: Update/create several assets
+  operationId: "patch_assets"
+  tags:
+    - Assets
+  x-versions:
+    - 2.1
+  x-ee: true
+  description: This endpoint allows you to update several assets at once.
+  x-body-by-line: Contains several lines, each line is a category in JSON standard format
```

Comment:
is an asset

Reply:
btw we are referring to the standard format, is it a good idea? As the get is not exactly the standard format